site/docs/components/progress/index.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Progress
+data:
+  description: "A Progress indicator gives the user an understanding of how long a system operation will take. It should be used when the operation will take more than a second to complete. Two variants are available to accommodate different layout constraints: linear and circular."
```

Comment:
The casing of "A Progress indicator" is a little weird, should it be "A progress indicator" or "Progress is an indicator"?

Reply:
True. I just copy-pasted from the Word doc the content team had reviewed. I'm guessing only "Progress" is capitalized since that's the name of the component. But, I think you're "Progress is an indicator..." suggestion might read better. Thoughts @jpmorganchase/salt-content ?

Reply:
Yeah tbh I copied this from toolkit. Either  "A progress indicator" or "Progress indicators give...." is better